tests/test_mocks.py

```diff
@@ -0,0 +1,79 @@
+"""Test mock scripts"""
+
+import logging
+import os
+from pathlib import Path
+from subprocess import call
+import unittest
+import numpy as np
+
+import pytest
+from pymchelper.estimator import Estimator
+
+
+from pymchelper.input_output import fromfile
+
+__MOCKS_PATH = Path(__file__).parent / "res" / "mocks"
```

Comment:
better to convert this to a fixture, instead of hardcoding a variable, same for `__EXPECTED_FLUKA_RESULTS`